### PR TITLE
NEW RELEASE: Update to R-4.0 and Bioconductor 3.11

### PR DESCRIPTION
**Pull request by nturaga**
This PR updates the images anvil-rstudio-base to base based on a new
image from the rocker group, i.e, rockerdev/tidyverse:4.0.0-ubuntu18.04.

The anvil-rstudio-bioconductor image inherits from anvil-rstudio-base
and includes R-4.0.0 and Bioconductor 3.11, along with some updated
packages.

The new set of docker images are:

1. us.gcr.io/anvil-gcr-public/anvil-rstudio-base:0.0.3

2. us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.5

Some changes made to files in README.md, VERSION in each of the
directories for the images to update the version number.

The CHANGELOG.md reflects the location and new "tag" of the docker
images in the google container registry

### CODE REVIEW

File: anvil-rstudio-bioconductor/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 | Latest Image Release | Docker Image URL |Date Updated | Questions or Feedback | 
 | --- | --- | --- | --- |
-| 0.0.3 | us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.3 | 12/11/2019 | Contact the [Bioconductor Team](mailto:nitesh.turaga@roswellpark.org) |
+| 0.0.4 | us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:0.0.4 | 05/08/2020 | Contact the [Bioconductor Team](mailto:nitesh.turaga@roswellpark.org) |
```
Comment (rtitle) [inline anvil-rstudio-bioconductor/README.md:3]: Should this be `0.0.5`? (missing entry for `0.0.4`?)

**Review by ekiernan — APPROVED** [review]
Updates to the README look good to me!

**Review by rtitle — APPROVED** [review]
Looks good to me -- if things are working feel free to merge and push to GCR.

Can we update Leo conf as well so this is tested? https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/resources/reference.conf#L10-L11

File: anvil-rstudio-bioconductor/README.md (hunk 1 above)
Comment (nturaga) [reply anvil-rstudio-bioconductor/README.md:3]: I think 0.0.5 is correct.

**Pull request merged**